extract/raw_array_cohort_extract.py

```diff
@@ -119,8 +114,8 @@ def get_subselect(fq_array_table, samples, id, extract_genotype_counts_only):
 
   sql = (
         f"CREATE OR REPLACE TABLE `{fq_destination_table}` \n"
-        f"PARTITION BY RANGE_BUCKET(probe_id, GENERATE_ARRAY(0, {MAX_PROBE_ID}, {PROBES_PER_PARTITION})) \n"
-        f"{FINAL_TABLE_TTL} "
+        f"PARTITION BY RANGE_BUCKET(probe_id, GENERATE_ARRAY(0, {number_of_partitions * probes_per_partition}, {probes_per_partition})) \n"
```

Comment:
I went back and forth on this.  Somebody has to know... so they could tell us the number of probes and partitions, and then we could calculate the partitions per probe, etc.  My thought was to leave it this way for now, and ultimately this could even be something extract from the dataset by querying probe_info.

Reply:
ok, sounds good